xrviz/utils.py

```diff
@@ -65,9 +67,21 @@ def callback(*events):
 
 
 def is_float(a):
-    # int, 'Nan' are also float
+    # int, 'Nan', 'inf' and '-inf' are also float
     try:
         float(a)
         return True
     except:
         return False
+
+if has_cartopy:
```

Comment:
Take away the `if` - this function simply does not get called ever if cartopy is missing